src/posix/platform/platform-posix.h

```diff
@@ -428,7 +462,21 @@ extern char gBackboneNetifName[IFNAMSIZ];
  *
  */
 extern unsigned int gBackboneNetifIndex;
-#endif
+
+#define otLogResultPlat(aError, aFormat, ...)                                          \
+    do                                                                                 \
+    {                                                                                  \
+        otError _err = (aError);                                                       \
+                                                                                       \
+        if (_err == OT_ERROR_NONE)                                                     \
+        {                                                                              \
+            otLogInfoPlat(aFormat ": %s", ##__VA_ARGS__, otThreadErrorToString(_err)); \
+        }                                                                              \
+        else                                                                           \
+        {                                                                              \
+            otLogCritPlat(aFormat ": %s", ##__VA_ARGS__, otThreadErrorToString(_err)); \
```

Comment:
I think `WARN` usually means the functionality is still working but with some restrictions. 
The log level should be `ERROR` because if it failed, the multicast routing is not working. 
But we don't have `ERROR`.

Reply:
I will fix to use WARN.